# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/Permissions.webidl
# Commit: f932a2a98670
# Full Hash: f932a2a9867024a760dabaa611a0bbe6936475b5
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 3: Screen Wake Lock permission integration. r=webidl,dom-core,smaug,pbz,edgar
#   
#   Depends on D189509
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189510
# ==============================================================================

diff -r 231b0b4efd16 -r f932a2a98670 dom/webidl/Permissions.webidl
--- a/dom/webidl/Permissions.webidl	Tue Dec 05 23:58:06 2023 +0000
+++ b/dom/webidl/Permissions.webidl	Tue Dec 05 23:58:06 2023 +0000
@@ -13,7 +13,8 @@
   "push",
   "persistent-storage",
   "midi",
-  "storage-access" // Defined in https://privacycg.github.io/storage-access/#permissions-integration
+  "storage-access", // Defined in https://privacycg.github.io/storage-access/#permissions-integration
+  "screen-wake-lock" // Defined in https://w3c.github.io/screen-wake-lock/
 };
 
 [GenerateInit]